internal/provider/resource_wlan.go

```diff
@@ -137,6 +138,18 @@ func resourceWLAN() *schema.Resource {
 				Optional:    true,
 				Default:     true,
 			},
+			"minimum_data_rate_2g": {
+				Description:  "Enable minimum data rate control for 2G devices",
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: validation.StringInSlice([]string{"1Mbps", "2Mbps", "5.5Mbps", "6Mbps", "9Mbps", "11Mbps", "12Mbps", "18Mbps", "24Mbps", "36Mbps", "48Mbps", "54Mbps"}, false),
+			},
+			"minimum_data_rate_5g": {
+				Description:  "Enable minimum data rate control for 5G devices",
+				Type:         schema.TypeString,
```

Comment:
I think it would be more user friendly to make these fields integers instead of strings. @paultyng what do you think?